src/components/CategorySection/CategoryContent.tsx

```diff
@@ -0,0 +1,91 @@
+import styled from '@emotion/styled';
+import { loading } from '@/components/common/Loading';
+import { ERROR_MESSAGES } from '@/constants/validation';
+import type { Category } from '@/types/category';
+
+interface Props {
+  themes: {
+    data: Category[] | null;
+    pending: boolean;
+    error: boolean;
+  };
+}
```

Comment:
```suggestion
interface Props {
  data: Category[] | null;
  pending: boolean;
  error: boolean;
}
```
이렇게 받아올 수 있도록 props를 정의해주면 어떨까요?

Reply:
이와 연계하여 props의 depth가 깊어지면 어떤 문제가 있을까요!?